### PR TITLE
Update version and sha.

### PR DESCRIPTION
**Pull request by LukeC92**
Update to version 0.8.2.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,10 +7,10 @@ package:
 source:
   fn: conda-execute-{{ version }}.tar.gz
   url: https://github.com/conda-tools/conda-execute/archive/v{{ version }}.tar.gz
-  sha256: c3182c0736fbfed07bc7fdd65e0331e62a440a3158b56cd8d0c5f249fd780489
+  sha256: 21388f4fd0e4e604b95231f207481ece08ba25cf76c6af63a94dcb63716c3c47
 
 build:
-  number: 2
+  number: 0
   script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (jakirkham) [inline recipe/meta.yaml]: Thoughts on making this a [`noarch: python`]( http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages ) package?

**Comment by djkirkham** [comment]
@conda-forge-admin, please rerender

**Comment by djkirkham** [comment]
ping @jakirkham @pelson

**Comment by pelson** [comment]
Looks good to me. If there are any issues that crop up (this is directed at everybody), let me know, and I'd be happy enough to revert / pull the binary.

**Comment by bjlittle** [comment]
@pelson Awesome, thanks!

**Comment by jakirkham** [comment]
Thanks everyone!

**Pull request merged**